frontend/config/deploy/staging.rb

```diff
@@ -1,9 +1,9 @@
-server '52.7.28.202', user: 'ubuntu', roles: %w{web app db}, primary: true
+server 'staging.resilienceatlas.org', user: 'ubuntu', roles: %w{web app db}, primary: true
 
 set :ssh_options, {
   forward_agent: true,
   auth_methods: %w(publickey)
 }
 
 set :node_env, 'production'
-set :branch, 'develop'
+set :branch, 'client/feature/RA2-37-fe-update-capistano'
```

Comment:
Why is this hard-coded?

Reply:
Will you create a separate file for production deployments?

Reply:
I guess that's for testing the script without merging to develop

Reply:
is hardcoded for testing purposes, it will be changed to `develop` before being merged. In production.rb file it will have a different setting. Should we setup the same changes in the production server?? I'll prefer to apply the changes in production when Tiago is available, just in case.